lib/central_mail/service.rb

```diff
@@ -7,7 +7,8 @@
 module CentralMail
   class Service < Common::Client::Base
     ################################################################
-    # Please do not use this module. It has been superceded by the #
+    # If you are not a Lighthouse API team, please do not use      #
+    # this module. It has been superceded by the                   #
```

Comment:
This comment was originally added by a different team than my own. I added "If you are not a Lighthouse API team" to the disclaimer since this service is still very much in use by the Lighthouse Decision Reviews and Benefits Intake APIs, and I didn't want someone to mistakenly get the idea that it's not in use anymore.